lib/comfortable_mexican_sofa/content/tags/mixins/file_content.rb

```diff
@@ -1,8 +1,27 @@
+#require 'app/helpers/comfy/cms_helper.rb'
+
 # frozen_string_literal: true
 
 # A mixin for tags that returns the file as their content.
 module ComfortableMexicanSofa::Content::Tag::Mixins
   module FileContent
+    def url_for_cms(blob)
+      url_helpers = Rails.application.routes.url_helpers
+
+      if Rails.env.test?
+        if blob.is_a?(ActiveStorage::Variant)
+          return url_helpers.rails_representation_path(blob, only_path: true)
+        else
+          return url_helpers.rails_blob_path(blob, only_path: true)
+        end
+      end
+
+      if blob.is_a?(ActiveStorage::Variant)
+        return url_helpers.rails_public_blob_url(blob.blob)
+      end
+
+      url_helpers.rails_public_blob_url(blob)
+    end
```

Comment:
This seems identical to https://github.com/Vivino/comfortable-mexican-sofa/pull/4/files#diff-1f97dd953c863b555f12982fd497859591e67dfa4089d2f5f004de300c1711dbR5-R19
Is it possible to delete one of them?

Reply:
unfortunately not possible. one is in the lib and we didn't find a way to share code

Reply:
I'd keep it as is and not invest more time into beutifying. we'll remove all this crap anyway

Reply:
All right, just wanted to make sure.